### ISSUE TITLE
training data does not has negative label value, but predict task has a lot of negative label , is it overfit?

### ISSUE THREAD

**Issue by hlee13** [Microsoft/LightGBM#900]
# config

task = train
boosting_type = gbdt
objective = regression_l1
metric = l2
metric_freq = 1
is_training_metric = true
max_bin = 255
data = data.train
valid_data = data.validate
num_trees = 200
learning_rate = 0.1
gaussian_eta = 0.9
num_leaves = 255
max_depth = 10
snapshot_freq = 50
tree_learner = data
feature_fraction = 0.9
bagging_freq = 5
bagging_fraction = 0.8
min_data_in_leaf = 100
min_sum_hessian_in_leaf = 5.0
is_enable_sparse = true
use_two_round_loading = true
is_save_binary_file = true
is_pre_partition = true
boost_from_average = false
output_model = data_model.txt
categorical_feature=1,2
num_machines = 4
local_listen_port = 12400
machine_list_file = mlist.txt

**Comment by guolinke** [comment]
@hlee13 
I think it is caused by "regerssion_l1". 
you can try the "regression" .

**Comment by hlee13** [comment]
@guolinke  regerssion_l1 cannot modify

**Comment by guolinke** [comment]
@hlee13 really? In many cases, "regreession" actually can get better l1 score than "regression_l1" .

**Comment by Laurae2** [comment]
@hlee13 L1 loss is unstable, please use a proxy metric (Fair loss, modified Hinge loss, etc.) to have the model converge better.
